src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -431,6 +438,46 @@ def get_extra_inputs_as_dict(args: argparse.Namespace) -> dict:
     return request_inputs
 
 
+def _parse_compare_args(subparsers) -> argparse.ArgumentParser:
+    compare = subparsers.add_parser(
+        "compare", help="Generate plots that compare multiple runs."
+    )
+    compare_group = compare.add_argument_group("Compare")
+    mx_group = compare_group.add_mutually_exclusive_group(required=False)
+    mx_group.add_argument(
+        "--config",
+        type=Path,
+        default=None,
+        help="The path to the YAML file that specifies plot configurations for "
+        "comparing multiple runs.",
+    )
+    mx_group.add_argument(
+        "-f",
+        "--files",
+        nargs="+",
+        default=[],
+        help="List of paths to the profile export JSON files. Users can specify "
+        "this option instead of the `--config` option if they would like "
+        "GenAI-Perf to generate default plots as well as initial YAML config file.",
+    )
+    compare.set_defaults(func=compare_handler)
+    return compare
+
+
+### Handlers ###
+
+
+def handler(args, extra_args):
+    from genai_perf.wrapper import Profiler
+
+    Profiler.run(args=args, extra_args=extra_args)
+
+
+def compare_handler(args: argparse.Namespace):
+    # TMA-1893: parse yaml file
+    pass
```

Comment:
Can you add a message that this is not yet supported if we are adding this? Are you going to work on this functionality as a PR off of this one or are you going to merge this into main first?

Reply:
This is all going into a dev branch that will add this support when complete.
I think this is fine if its not going into main. In main, i agree 100%

Reply:
This is going into `compare-subcommand` dev branch and not directly into the main branch. And I am working on TMA-1893 that will fill this part up.

Reply:
Got it, thanks!